charts/node-analyzer/values.yaml

```diff
@@ -106,6 +106,14 @@ nodeAnalyzer:
       key: node-role.kubernetes.io/master
     - effect: NoSchedule
       key: node-role.kubernetes.io/control-plane
+    - effect: NoSchedule
+      key: node-role.kubernetes.io/controlplane
```

Comment:
question: how does rancher know which key (controlplane or control-plane) to pick?

Reply:
kubeadm adds a taint `node-role.kubernetes.io/control-plane`, rancher instead uses `node-role.kubernetes.io/controlplane` by default, adding a toleration for both is the easiest way to allow the agent and other pods to be scheduled on those nodes

Reply:
I just read up on taints and tolerations and this starts to make sense. thank you!